woodwork/tests/utils/test_utils.py

```diff
@@ -447,3 +448,9 @@ def test_col_is_datetime():
     for input, expected in list(zip(inputs, expected_values)):
         actual = col_is_datetime(input)
         assert actual is expected
+
+
+def test_infer_datetime_format(datetimes):
+    for series in datetimes:
+        fmt = _infer_datetime_format(series)
+        assert fmt == '%m/%d/%Y'
```

Comment:
Does this also work reliably if there is a time included? Might want to test for that too.